_includes/com-config/com-config-advertise-extra.md

```diff
@@ -0,0 +1,5 @@
+### `advertise` and `advertise-grpc` additional
```

Comment:
Only used once. Does this have a plan to be re-used?

Reply:
Yep, so this will go into other featurebase-config doco as it's written.